pkg/proto/datadog/api/README.md

```diff
@@ -0,0 +1,57 @@
+## gRPC: Protobuf and Gateway code generation 
+
+To generate the code for the API you have defined in your `.proto`
+files we will need three different grpc-related packages: 
+
+- protobuf - protoc-gen-go: generates the golang protobuf definitions.
+- grpc-gateway - protoc-gen-grpc-gateway: generates the gRPC-REST gateway  
+- grpc-gateway - protoc-gen-swagger (optional)
+
+### Install
+
+Run the following:
+```
+go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger github.com/golang/protobuf/protoc-gen-go
+```
+This should drop all required binaries in your `$GOPATH/bin`
+
+Remember to make sure `GOPATH/bin` is in your `PATH`, also make
+sure no other versions of those binaries you may have installed
+elsewhere take precedence (`which` is your friend).
+
+### Code Generation
+
+Chdir yourself into this directory (`cmd/agent/api/pb`), and run
```

Comment:
Wrong path

Reply:
I need to cleanup the entire README, thanks for pointing this out.